sdk/documentintelligence/azure-ai-documentintelligence/samples/aio/sample_manage_models_async.py

```diff
@@ -0,0 +1,152 @@
+        DocumentBuildMode,
+        BuildDocumentModelRequest,
+        AzureBlobContentSource,
+    )
+    from azure.core.credentials import AzureKeyCredential
+
+    endpoint = os.environ["DOCUMENTINTELLIGENCE_ENDPOINT"]
+    key = os.environ["DOCUMENTINTELLIGENCE_API_KEY"]
+    container_sas_url = os.environ["DOCUMENTINTELLIGENCE_STORAGE_CONTAINER_SAS_URL"]
+
+    document_model_admin_client = DocumentIntelligenceAdministrationClient(
+        endpoint, AzureKeyCredential(key)
+    )
+    async with document_model_admin_client:
+        poller = await document_model_admin_client.begin_build_document_model(
+            BuildDocumentModelRequest(
+                model_id=str(uuid.uuid4()),
+                build_mode=DocumentBuildMode.TEMPLATE,
+                azure_blob_source=AzureBlobContentSource(container_url=container_sas_url),
+                description="my model description",
+            )
+        )
+        model = await poller.result()
+
+        print(f"Model ID: {model.model_id}")
+        print(f"Description: {model.description}")
+        print(f"Model created on: {model.created_date_time}")
+        print(f"Model expires on: {model.expiration_date_time}")
+        print("Doc types the model can recognize:")
+        for name, doc_type in model.doc_types.items():
+            print(
+                f"Doc Type: '{name}' built with '{doc_type.build_mode}' mode which has the following fields:"
+            )
+            for field_name, field in doc_type.field_schema.items():
+                print(
+                    f"Field: '{field_name}' has type '{field['type']}' and confidence score "
+                    f"{doc_type.field_confidence[field_name]}"
+                )
+        # [END build_model]
+
+        # [START get_resource_info]
+        account_details = await document_model_admin_client.get_resource_info()
+        print(
+            f"Our resource has {account_details.custom_document_models.count} custom models, "
+            f"and we can have at most {account_details.custom_document_models.limit} custom models"
+        )
+        neural_models = account_details.custom_neural_document_model_builds
+        print(
+            f"The quota limit for custom neural document models is {neural_models.quota} and the resource has"
+            f"used {neural_models.used}. The resource quota will reset on {neural_models.quota_reset_date_time}"
+        )
+        # [END get_resource_info]
+
+        # Next, we get a paged list of all of our custom models
+        # [START list_models]
+        models = document_model_admin_client.list_models()
+
+        print("We have the following 'ready' models with IDs and descriptions:")
+        async for model in models:
+            print(f"{model.model_id} | {model.description}")
+        # [END list_models]
+
+        # let's build a model to use for this sample
+        poller = await document_model_admin_client.begin_build_document_model(
```

Comment:
Nit: building another model here although you already created one at the beginning of the sample

Reply:
Removed the second build.